templates/navbar.html

```diff
@@ -1,18 +1,24 @@
 <!-- Navbar -->
-<nav class="navbar fixed-top navbar-expand-lg navbar-dark bg-primary">
+<nav class="navbar fixed-top navbar-expand-lg navbar-dark bg-primary" style="padding: 0 0">
```

Comment:
You can’t remove the padding via a style tags here, it removes the padding on mobile.